### ISSUE TITLE
APE metrics and method name

### ISSUE THREAD

**Issue by Mathux** [anindita127/Complextext2animation#2]
Hello,
First of all, thanks for releasing your code.

In your paper, in Figure 4, the APE metrics looks different from the Table 1 of Languag2Pose (https://arxiv.org/pdf/1907.01108.pdf). Is the evaluation script different, or are you using different train/test split?

And by the way, how should we name your method?

Thanks,

**Comment by anindita127** [comment]
Hi, thanks for your interest in our work. I have used the pre-trained model provided in language2pose repo to evaluate the baseline metrics. However, they didn't provide the code for evaluation so it's unclear if it is something different from how we have calculated APE. I have used the same train/test split as language2pose.

You can refer to our work as Ghosh et al. since we haven't come up with a short name for the method. Hope these answers your questions.

**Comment by Mathux** [comment]
I have another question about evaluation. In the paper, you say:
> We randomly split the dataset in the ratio of 0.6, 0.2, and 0.2 for our training, validation, and test sets.

Are these the same splits as in Language2Pose? (As you compare with their pre-trained model).
In any case, where or how can I download the splits?

Thanks again

**Comment by anindita127** [comment]
Yes, I have used the same splits as language2pose. You can do so by keeping the seed value unchanged. The splits are generated when you load the data. The code is in dataUtils.py.

**Comment by Mathux** [comment]
Hello again,

When I generated the visuals with your code, I noticed that, for the test set, there are 520 videos. When I do the same process with language2pose, I obtain 587 videos. It seems that the 520 videos is a subset of the 587 videos. Do you know why some videos are missing?

Thanks for your help
